### PR TITLE
[IMP] point_of_sale: add payment methods image

### PR DESCRIPTION
**Pull request by rhe-odoo**
This commit adds default image to payment methods. It also let the user set custom images to his payment methods. Those images will be displayed in the payment screen next to the name.

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/scss/pos.scss [hunk 1]
```diff
@@ -2375,10 +2374,22 @@ td {
     flex-grow: 1;
     flex-basis: 100%;
 }
-.paymentmethod .payment-name {
+.paymentmethod .payment-method-display {
     flex-grow: 1;
```
Comment (trj-odoo) [inline addons/point_of_sale/static/src/scss/pos.scss:2378]:
```css
display: flex;
align-items: center;
```

File: addons/point_of_sale/static/src/xml/Screens/PaymentScreen/PaymentScreen.xml [hunk 2]
```diff
@@ -28,8 +28,9 @@
                                    t-key="paymentMethod.id">
                                     <div class="button paymentmethod"
                                         t-on-click="() => this.trigger('new-payment-line', paymentMethod)">
-                                        <div class="payment-name">
-                                            <t t-esc="paymentMethod.name" />
+                                        <div class="payment-method-display">
+                                            <span><img class="payment-method-icon" t-att-src="paymentMethodImage(paymentMethod.id)"/></span>
```
Comment (trj-odoo) [inline addons/point_of_sale/static/src/xml/Screens/PaymentScreen/PaymentScreen.xml:32]:
```suggestion
                                            <div class=payment-method-icon-container><img class="payment-method-icon" t-att-src="paymentMethodImage(paymentMethod.id)"/></div>
```
```css
.payment-method-icon-container {
    height: 50px;
    width: 50px;
}

**Comment by trj-odoo** [comment]
@robodoo r+

**Comment by d-fence** [comment]
robodoo check

**Comment by trj-odoo** [comment]
@robodoo r+

**Pull request closed without merging**